onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -752,6 +761,18 @@ def _where_input_wrangler(
     xfail(
         "round", variant_name="decimals_neg_3", reason="The op does not support decimals yet"
     ),
+    xfail(
+        "scatter_reduce",
+        variant_name="mean",
+        reason="ORT doesn't support reduce='mean' option",
+        test_class_name="TestOutputConsistencyEager",
+    ),
+    xfail(
+        "scatter_reduce",
+        variant_name="mean",
+        reason="ORT doesn't support reduce='mean' option",
+        test_class_name="TestOutputConsistencyFullGraph",
+    ),
```

Comment:
```suggestion
    xfail(
        "scatter_reduce",
        variant_name="mean",
        reason="ORT doesn't support reduce='mean' option",
    ),
```

to enable it for both tests